pages/index.tsx

```diff
@@ -8,8 +8,36 @@ import { montserrat } from "@/utils/fonts";
 import Link from "next/link";
 import { Testimonials } from "../components/testimonials";
 import { CalendarEvents } from "../components/home/calendar-events";
+import dayjs from "dayjs";
+import advancedFormat from "dayjs/plugin/advancedFormat";
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
i can't find where you're using this import

Reply:
We don't need this import in retrospect, I'll remove it